.github/workflows/ci_dockers.yml

```diff
@@ -52,22 +52,23 @@ jobs:
           push: false
         timeout-minutes: 40
 
-  build-cuda:
-    runs-on: ubuntu-20.04
-    strategy:
-      fail-fast: false
-      matrix:
-        python_version: [3.7]
-        pytorch_version: [1.5]
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: Publish Master to Docker
-        # publish master
-        uses: docker/build-push-action@v1.1.0
-        with:
-          dockerfile: dockers/base-cuda/Dockerfile
-          build_args: PYTHON_VERSION=${{ matrix.python_version }},PYTORCH_VERSION=${{ matrix.pytorch_version }}
-          push: false
-        timeout-minutes: 40
+# TODO: uncomment this with fixing CUDA docker, no need to increase mergify count
```

Comment:
why leave the comment if its in the history?

Reply:
it is another PR #3074